mgmtd/mgmt_ds.h

```diff
@@ -354,6 +354,29 @@ extern void mgmt_ds_dump_tree(struct vty *vty, struct mgmt_ds_ctx *ds_ctx,
 extern int mgmt_ds_dump_ds_to_file(char *file_name,
 				   struct mgmt_ds_ctx *ds_ctx);
 
+/*
+ * Set (or reset) automatic saving of contents of 'running'
+ * datastore to 'startup' datastore.
+ *
+ * set
+ *    Set (or reset) the setting.
+ *
+ * Returns:
+ *    None.
+ */
+extern void mgmt_ds_set_autosave_config_to_startup(bool set);
+
+/*
+ * Determine if automatic saving of contents of 'running'
+ * datastore to 'startup' datastore is turned on or not.
+ *
+ * Returns:
+ *    True if automatic saving of contents of 'running'
+ *    datastore to 'startup' datastore is turned on. Else
+ *    False.
+ */
+extern bool mgmt_ds_autosave_config_to_startup(void);
+
```

Comment:
Can I please still keep it? If you will insist it one more time I will remove it. But I like it the way it is currently. Otherwise I will need to import a pointer into mgmt_vty.c like the way I had to do for other sub-modules.

Reply:
What is wrong with a daemon accessing it's own global value? Using getter and setter here doesn't help anything IMO, it's just bloated abstraction that obfuscates things.